codeflash/cli_cmds/cmd_init.py

```diff
@@ -587,6 +587,11 @@ def configure_pyproject_toml(setup_info: SetupInfo) -> None:
         )
     elif formatter == "don't use a formatter":
         formatter_cmds.append("disabled")
+    if formatter in ["black", "ruff"]:
+        try:
+            result = subprocess.run([formatter], capture_output=True, check=False)
```

Comment:
nit: shouldnt we use shutil.which to check if its in file path rather than running the process?
And we might have to test this in other OS for consistency.

Reply:
we want to test for existence the same way it is run later for formatting. To reduce any inconsistencies